endoreg_db/models/media/pdf/raw_pdf.py

```
@@ -497,23 +516,28 @@ def verify_existing_file(self, fallback_file):
+            if not _file.field.storage.exists(_file.name):
+                logger.warning(f"File missing at storage path {_file.name}. Attempting copy from fallback {fallback_file}")
                 if fallback_file.exists():
                     with fallback_file.open("rb") as f:
                         # Use save method which handles storage backend
-                        self.file.save(Path(self.file.name).name, File(f), save=True)  # Re-save the file content
-                    logger.info(f"Successfully restored file from fallback {fallback_file} to {self.file.name}")
+                        _file.save(Path(_file.name).name, File(f), save=True)  # Re-save the file content
+                    logger.info(f"Successfully restored file from fallback {fallback_file} to {_file.name}")
                 else:
                     logger.error(f"Fallback file {fallback_file} does not exist.")
         except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)